extensions/objects/templates/CoordTwoDimEditor.js

```diff
@@ -17,81 +17,54 @@ oppia.directive('coordTwoDimEditor', [
   function(UrlInterpolationService, OBJECT_EDITOR_URL_PREFIX) {
     return {
       controller: ['$scope', '$timeout', function($scope, $timeout) {
-        $scope.schemaLatitude = {
-          type: 'float',
-          validators: [{
-            id: 'is_at_least',
-            min_value: -90.0
-          }, {
-            id: 'is_at_most',
-            max_value: 90.0
-          }]
-        };
-
-        $scope.schemaLongitude = {
-          type: 'float',
-          validators: [{
-            id: 'is_at_least',
-            min_value: -180.0
-          }, {
-            id: 'is_at_most',
-            max_value: 180.0
-          }]
-        };
-
-        var updateMarker = function(lat, lng) {
-          var latLng = new google.maps.LatLng(lat, lng);
-
-          $timeout(function() {
-            if ($scope.mapMarker) {
-              $scope.mapMarker.setPosition(latLng);
-            } else {
-              $scope.mapMarker = new google.maps.Marker({
-                map: $scope.map,
-                position: latLng
-              });
+        angular.extend($scope, {
+          center: {
+            lat: $scope.value[0],
+            lng: $scope.value[1],
+            zoom: 0
+          },
+          markers: {
+            markersmarkers: {
```

Comment:
Just checking, is this a typo ("markersmarkers")? Seems like a weird key name.

Reply:
I had found this somewhere in examples, although the correct name is mainMarker but it was working with this also, don't know how ui-leaflet handle this. Changed!